cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/util/AwsCloudFormationErrorMessageProvider.java

```diff
@@ -34,10 +38,13 @@ public String getErrorReason(AuthenticatedContext ac, String stackName, Resource
     }
 
     public String getErrorReason(AwsCredentialView credentialView, String region, String stackName, ResourceStatus resourceErrorStatus) {
+        LOGGER.debug("Getting error reason in Cloudformation stack {} for resources with {} status", stackName, resourceErrorStatus);
+
         AmazonCloudFormationClient cfClient = awsClient.createCloudFormationClient(credentialView, region);
         DescribeStacksRequest describeStacksRequest = new DescribeStacksRequest().withStackName(stackName);
         DescribeStacksResult describeStacksResult = cfClient.describeStacks(describeStacksRequest);
```

Comment:
can we add one more line to print the whole describeStacksResult?

Reply:
sure, added it